test/mocks/cache.js

```diff
@@ -1,8 +1,10 @@
 var MockCache = function(opt_store) {
   this.store = opt_store || {};
+  this.counter = 0;
 };
 
 MockCache.prototype.get = function(key) {
+  this.counter++;
```

Comment:
Nit: Use prefix `++this.counter;` as a general style (I don't see it in Google's Javascript guide, but it's in Google's C++ guide)
More discussion: https://softwareengineering.stackexchange.com/questions/59880/avoid-postfix-increment-operator

Reply:
Done